### PR TITLE
Variable deposit slider

### PR DESCRIPTION
**Pull request by srleach**
## Features

- Update Variable Deposit slider to use better implementation.

### CODE REVIEW

**Comment by srleach** [comment]
@EvanBarbour3

File: public/js/custom-deposit.main.js [hunk 1]
```diff
@@ -20,6 +48,9 @@ function depositValueHasChanged(changedElement){
         $(this).val(depositValueWithinRange(changedElement));
     });
 
+    console.log($(changedElement).parent().parent());
```
Comment (srleach) [inline public/js/custom-deposit.main.js]: Shouldn't be here

File: public/css/nouislider.css [hunk 2]
```diff
@@ -0,0 +1,165 @@
+
+/* Functional styling;
+ * These styles are required for noUiSlider to function.
+ * You don't need to change these rules to apply your design.
+ */
+.noUi-target,
+.noUi-target * {
+-webkit-touch-callout: none;
```
Comment (wnowicki) [inline public/css/nouislider.css]: - keep it aligned

File: public/css/nouislider.css [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+	position: relative;
+	z-index: 1;
+}
+.noUi-stacking .noUi-handle {
+/* This class is applied to the lower origin when
+   its values is > 50%. */
+	z-index: 10;
+}
+.noUi-state-tap .noUi-origin {
+-webkit-transition: left 0.3s, top 0.3s;
```
Comment (wnowicki) [inline public/css/nouislider.css]: - the same

File: public/css/nouislider.css (hunk 2 above)
Comment (srleach) [inline public/css/nouislider.css]: Library asset

File: public/css/nouislider.css (hunk 3 above)
Comment (srleach) [inline public/css/nouislider.css]: Library asset

**Comment by EvanBarbour3** [comment]
@srleach Just the slider amount on the top. I'd suggest having a <p> element or a <br>, and have the class `hidden-md hidden-lg`, so that it will display on smaller devices, leaving a gap. Unless the library has a solution for it

**Comment by srleach** [comment]
Suggestion is good, but unfortunately not in the principals of modern design. We have media queries to be able to selectively apply rules to content. HTML should be clean, concise and self descriptive. I.e not containing dummy, hidden elements for spacing. That's what the CSS is for 👍

**Comment by srleach** [comment]
@EvanBarbour3 

```css
@media screen and (max-width: 767px) {
            .deposit-slider-container {
                margin-top: 36px;
            }
            .deposit-container {
                padding: 20px 0 50px 0;
            }
        }
```

👍

**Comment by EvanBarbour3** [comment]
@srleach Can see it is working. @wnowicki

**Pull request merged**